### ISSUE TITLE
[BUG]:  input.__PrivateStripeElement-input ARIA hidden element must not be focusable or contain focusable elements

### ISSUE THREAD

**Issue by edfenergy-kateb** [stripe/react-stripe-js#441]
### What happened?

When running an accessibility scan on a page that uses the following @stripe/react-stripe-js components `<CardNumberElement>`, `<CardExpiryElement> and `<CardCvcElement>` 3 Serious accessibility violations flag up - "ARIA hidden element must not be focusable or contain focusable elements". 

The element this pertains to is `<input.__PrivateStripeElement-input/>`

The fields are hidden to screen readers however the input field is tab-able when navigating with a keyboard.

Screenshot taken from the accessibility scan

As the element is Private i'm unable to customise. 

<img width="712" alt="Screenshot 2023-09-07 at 07 15 39" src="https://github.com/stripe/react-stripe-js/assets/129743436/64cf6c40-beb0-4300-984d-22d1086f98b1">

**Comment by brendanm-stripe** [comment]
Hey @edfenergy-kateb - thanks for reporting. This class of reported issues is a known false positive due to the way Stripe.js & Elements work. Those hidden inputs are necessary to manage focus within the secure iframe(s), and accessible functionality is managed via related dom elements.
